src/lib/components/proposals/ProposalSelector.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+    import InputSelect from '$lib/components/ui/InputSelect.svelte';
+
+    // These types correspond with IC Proposal Documentation (https://wiki.internetcomputer.org/wiki/Network_Nervous_System#Proposals)
+    export type ProposalType = 'Motion' | 'AddOrRemoveNodeProvider';
```

Comment:
- can you move this type to `governance.ts` in `types`
- can you rename it to `ProposalAction` - i.e. in Candid it's an `Action` enum